src/main/java/race/CarRaceGameHistory.java

```diff
@@ -20,7 +24,39 @@ public int getRound() {
         return this.round;
     }
 
-    public static CarRaceGameHistory record(int round, int step) {
-        return new CarRaceGameHistory(round, step);
+    public String getName() {
+        return this.name;
+    }
+
+    public static CarRaceGameHistory record(int round, int step, String name) {
+        return new CarRaceGameHistory(round, step, name);
+    }
+
+    public static List<CarRaceGameHistory> selectWinningHistories(List<CarRaceGameHistory> histories) {
+        if (isNotSameRound(histories)) {
+            throw new IllegalArgumentException("같은 라운드의 게임 기록이 아닙니다.");
+        }
+
+        int maxStep =
```

Comment:
stream 사용시 적절한 개행을 추가해보시면 좋을것 같아요!

```suggestion
        int maxStep = histories.stream()
            .mapToInt(CarRaceGameHistory::getStep)
            .max()
            .orElseThrow(NoSuchElementException::new);
```

Reply:
넵 수정했습니다